components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -32,13 +40,17 @@ typealias OnNeedToRequestPermissions = (permissions: Array<String>) -> Unit
  * Once the dialog is closed the [PermissionRequest] will be consumed.
  *
  * @property anchorView the view which the prompt is going to be anchored.
+ * @param iconsTintColorResource Optional ID of the color resource to tint the icons.
  * @property sessionManager the [SessionManager] instance in order to subscribe
  * to the selected [Session].
  * @property onNeedToRequestPermissions a callback invoked when permissions
  * need to be requested. Once the request is completed, [onPermissionsResult] needs to be invoked.
  **/
+
+@Suppress("TooManyFunctions")
 class SitePermissionsFeature(
     private val anchorView: View,
+    @ColorRes private val iconsTintColorResource: Int = android.R.color.black,
```

Comment:
I wonder if we can find a better solution here: Maybe start with just using the same tint as the theme text color? Overall this looks like a good candidate for a theme/style since we want to style **ALL** instances and not pass in a customized color for every single prompt.